src/tests/Microsoft.PowerFx.Connectors.Tests/PowerPlatformConnectorTests.cs

```diff
@@ -16,23 +17,36 @@ namespace Microsoft.PowerFx.Tests
     public class PowerPlatformConnectorTests : PowerFxTest
     {
         // Exercise calling the MSNWeather connector against mocked Swagger and Response.json. 
-        [Fact]
-        public async Task MSNWeatherConnector_CurrentWeather()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task MSNWeatherConnector_CurrentWeather(bool useSwaggerParameter)
         {
             using var testConnector = new LoggingTestServer(@"Swagger\MSNWeather.json");
             var apiDoc = testConnector._apiDocument;
             var config = new PowerFxConfig();
 
             using var httpClient = new HttpClient(testConnector);
-            using var client = new PowerPlatformConnectorClient(
-                "firstrelease-001.azure-apim.net", // endpoint
-                "839eace6-59ab-4243-97ec-a5b8fcc104e4", // environment
-                "shared-msnweather-8d08e763-937a-45bf-a2ea-c5ed-ecc70ca4", // connectionId
-                () => "AuthToken1",
-                httpClient)
-            {
-                SessionId = "MySessionId"
-            };
+
+            using var client = useSwaggerParameter ?
+                new PowerPlatformConnectorClient(
+                    apiDoc,                                                     // Swagger file
+                    "839eace6-59ab-4243-97ec-a5b8fcc104e4",                     // environment
+                    "shared-msnweather-8d08e763-937a-45bf-a2ea-c5ed-ecc70ca4",  // connectionId
+                    () => "AuthToken1",
+                    httpClient)
+                {
+                    SessionId = "MySessionId"
+                }
+                : new PowerPlatformConnectorClient(
+                    "firstrelease-001.azure-apim.net",                          // endpoint
```

Comment:
We're not allowing https:// for the endpoint. 
Can we add a test to ensure we get the header below correct even when endpoint has https:// ?  (the header doesn't have https)

authority: firstrelease-001.azure-apim.net